tests/foreman/cli/test_import.py

```diff
@@ -391,19 +391,11 @@ def gen_import_chost_data():
                 u'organization': gen_string('alphanumeric'),
             } for i in range(len(org_ids))],
             u'custom-channels': [{
-                u'key': u'org_id',
+                u'key': u'channel_id',
                 u'key_id': type(u'')(i + 1),
                 u'channel_name': gen_string('alphanumeric'),
                 u'channel_label': gen_string('alphanumeric')}
-                for i in range(len(org_ids))
-            ],
-            # wipe all channel labels to make hammer skip the sync
-            u'channels': [{
-                u'key': u'channel_id',
-                u'key_id': type(u'')(i),
-                u'channel_label': u'',
-                u'channel_name': gen_string('alphanumeric')}
-                for i in set(range(101, 113))
+                for i in range(106, 113)
```

Comment:
what does these numbers mean? why 106 or 113? That information has to be highlighted in the code